fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1651,15 +1669,18 @@ ACTOR Future<SrcDestTeamPair> getSrcDestTeams(DDQueueData* self,
 	return {};
 }
 
-ACTOR Future<Void> BgDDLoadRebalance(DDQueueData* self, int teamCollectionIndex, int ddPriority) {
+ACTOR Future<Void> BgDDLoadRebalance(DDQueueData* self,
+                                     int teamCollectionIndex,
+                                     int ddPriority,
```

Comment:
Can we eliminate the use of `ddPriority` here? One way would be to build a map of DataMovementReason to int which has the priority mapping. During DD startup, it could be initialized with server knob values. For now, maybe we could have a function with a switch statement that returns the priority value corresponding to each enum.

Reply:
I've used a func called mapMovementReasonToKnobPriority to give the one-to-one relation from MovementReason to Priority